draft-ietf-acme-acme.md

```diff
@@ -1996,14 +2020,28 @@ authorization object is implicitly tied to the account key used to sign the
 request.
 
 When a client receives an order from the server it downloads the authorization
-resources by sending GET requests to the indicated URLs.  If the client
+resources by sending POST-as-GET requests to the indicated URLs.  If the client
 initiates authorization using a request to the new authorization resource, it
 will have already received the pending authorization object in the response
 to that request.
 
 ~~~~~~~~~~
-GET /acme/authz/1234 HTTP/1.1
+POST /acme/authz/1234 HTTP/1.1
 Host: example.com
+Content-Type: application/jose+json
+Accept: application/pkix-cert
+
+{
+  "protected": base64url({
+    "alg": "ES256",
+    "kid": "https://example.com/acme/acct/1",
+    "nonce": "uQpSjlRb4vQVCjVYAyyUWg",
+    "url": "https://example.com/acme/authz/1234",
+    "typ": "GET"
+  }),
+  "payload": base64url({}),
```

Comment:
Shouldn't this be `"payload": null,` now since its replacing a GET to this authz with a POST-as-GET?

Reply:
I think this should actually be `"payload": ""`, since JSON distinguishes between an empty string (or equivalently, zero-length payload) and the null value.